`test_start_kytos_api_core` failing

The test is assuming that the NApp list will always come in the same order, which is not guaranteed by the API. The test can be fixed by casting both lists to Python Sets.

```python
_________________ TestE2EKytosServer.test_start_kytos_api_core _________________

self = <tests.test_e2e_01_kytos_startup.TestE2EKytosServer testMethod=test_start_kytos_api_core>

    def test_start_kytos_api_core(self):
        # check server status if it is UP and running
        api_url = KYTOS_API+'/core/status/'
        response = requests.get(api_url)
        self.assertEqual(response.status_code, 200)
        data = response.json()
        self.assertEqual(data['response'], 'running')
    
        # check the list of enabled napps
        expected_napps = [
                ["kytos", "pathfinder"],
                ["kytos", "mef_eline"],
                ["kytos", "storehouse"],
                ["kytos", "flow_manager"],
                ["kytos", "of_core"],
                ["kytos", "topology"],
                ["kytos", "of_lldp"]
            ]
        api_url = KYTOS_API+'/core/napps_enabled/'
        response = requests.get(api_url)
        self.assertEqual(response.status_code, 200)
        data = response.json()
>       self.assertEqual(data['napps'], expected_napps)
E       AssertionError: Lists differ: [[u'kytos', u'storehouse'], [u... != [['kytos', 'pathfinder'], ['ky...
E       
E       First differing element 0:
E       [u'kytos', u'storehouse']
E       ['kytos', 'pathfinder']
E       
E       First list contains 1 additional elements.
E       First extra element 7:
E       [u'kytos', u'mef_eline']
E       
E       - [[u'kytos', u'storehouse'],
E       -  [u'kytos', u'pathfinder'],
E       ? - ^         -
E       
E       + [['kytos', 'pathfinder'],
E       ?  ^
E       
E       -  [u'kytos', u'maintenance'],
E       -  [u'kytos', u'of_core'],
E       ?   -         - ^  ^^^
E       
E       +  ['kytos', 'mef_eline'],
E       ?             ^^  ^^^^
E       
E       +  ['kytos', 'storehouse'],
E       -  [u'kytos', u'topology'],
E       -  [u'kytos', u'of_lldp'],
E       -  [u'kytos', u'flow_manager'],
E       ?   -         -
E       
E       +  ['kytos', 'flow_manager'],
E       +  ['kytos', 'of_core'],
E       +  ['kytos', 'topology'],
E       -  [u'kytos', u'mef_eline']]
E       ?   -         - ^^  - ^^^
E       
E       +  ['kytos', 'of_lldp']]
E       ?             ^   ^^^

tests/test_e2e_01_kytos_startup.py:49: AssertionError
```

--- a/tests/test_e2e_01_kytos_startup.py
+++ b/tests/test_e2e_01_kytos_startup.py
@@ -34,19 +34,21 @@ class TestE2EKytosServer(unittest.TestCase):
 
         # check the list of enabled napps
         expected_napps = [
-                ["kytos", "pathfinder"], 
-                ["kytos", "mef_eline"], 
-                ["kytos", "storehouse"], 
-                ["kytos", "flow_manager"], 
-                ["kytos", "of_core"], 
-                ["kytos", "topology"], 
-                ["kytos", "of_lldp"]
+                ("kytos", "pathfinder"),
+                ("kytos", "mef_eline"),
+                ("kytos", "maintenance"),
+                ("kytos", "storehouse"),
+                ("kytos", "flow_manager"),
+                ("kytos", "of_core"),
+                ("kytos", "topology"),
+                ("kytos", "of_lldp")
             ]
         api_url = KYTOS_API+'/core/napps_enabled/'
         response = requests.get(api_url)
         self.assertEqual(response.status_code, 200)
         data = response.json()
-        self.assertEqual(data['napps'], expected_napps)
+        #self.assertEqual(set(data['napps']), set(expected_napps))
+        assert set([tuple(lst) for lst in data['napps']]) == set(expected_napps)
 
         # check disable a napp
         api_url = KYTOS_API+'/core/napps/kytos/mef_eline/disable'
@@ -56,14 +58,20 @@ class TestE2EKytosServer(unittest.TestCase):
         response = requests.get(api_url)
         self.assertEqual(response.status_code, 200)
         data = response.json()
-        self.assertEqual(data['napps'], expected_napps[:1] + expected_napps[2:])
+        #self.assertEqual(data['napps'], expected_napps[:1] + expected_napps[2:])
+        assert set([tuple(lst) for lst in data['napps']]) == set(expected_napps) - set([("kytos", "mef_eline")])
 
+        # restart kytos and check if the switches are still enabled
+        self.net.start_controller(clean_config=False)
         self.net.wait_switches_connect()
+
         api_url = KYTOS_API+'/core/napps_enabled/'
         response = requests.get(api_url)
         self.assertEqual(response.status_code, 200)
         data = response.json()
-        self.assertEqual(data['napps'], expected_napps[:1] + expected_napps[2:])
+        #self.assertEqual(data['napps'], expected_napps[:1] + expected_napps[2:])
+        assert set([tuple(lst) for lst in data['napps']]) == set(expected_napps) - set([("kytos", "mef_eline")])
+
 
         # check enable a napp
         api_url = KYTOS_API+'/core/napps/kytos/mef_eline/enable'
@@ -73,7 +81,8 @@ class TestE2EKytosServer(unittest.TestCase):
         response = requests.get(api_url)
         self.assertEqual(response.status_code, 200)
         data = response.json()
-        self.assertEqual(data['napps'], expected_napps)
+        #self.assertEqual(data['napps'], expected_napps)
+        assert set([tuple(lst) for lst in data['napps']]) == set(expected_napps)
 
         # test auth api
         # TODO

